script.js

```diff
@@ -0,0 +1,73 @@
+function setBackgroundColor(heightScreen) {
+  if (heightScreen > 1000) {
+    $("#home-page").css("background-image", "none");
+    $("#home-page").css("background-color", "rgb(133, 57, 98)");
```

Comment:
Note: you can pass an object to the jquery css function to update multiple styles at the same time with a single call ([docs](http://api.jquery.com/css/#css-properties)), e.g.:

```js
$('#home-page').css({
  'backgroud-image': 'none',
  'background-color': 'rgb(133, 57, 98)'
});
```